scripts/build_demand_profiles.py

```diff
@@ -74,7 +104,7 @@ def get_load_paths_gegis(ssp_parentfolder, config):
     ["/data/ssp2-2.6/2030/era5_2013/Africa.nc", "/data/ssp2-2.6/2030/era5_2013/Africa.nc"]
     """
     countries = config.get("countries")
-    region_load = getContinent(countries)
+    region_load = get_gegis_regions(countries)
```

Comment:
Shouldn't we also remove `getContinent()` from `_helpers`? As far as I can see, it isn't used anywhere else.

Reply:
Agree :) Removing!